pkg/provision/converter_test.go

```diff
@@ -38,12 +38,12 @@ func TestTfgridReservationContainer1_ToProvisionType(t *testing.T) {
 		NodeID            string
 		Flist             string
 		HubURL            string
-		Environment       map[string]string
+		Environment       map[string]interface{}
```

Comment:
Why this type change ? environment has always value as string

Reply:
I know, but because the schema limitation, we can only generate map[string]interface{} (there is no way in schema to define type of the value of the map!)

Note: Actually i support that in our schema pkg, but it's not compatible with jsx